generate-requirements-full.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+docker run \
+  -v $(pwd):/src \
+  -ti --rm civisanalytics/datascience-python:7 \
```

Comment:
To ensure consistency, we probably should need some sort of a unit test to assert that "the ds-py image tag in this bash script here == the ds-py image tag in `Dockerfile` of this repo", which means we would need `7.0.0` here